src/locales/en/messages.json

```diff
@@ -3417,6 +3417,9 @@
   "emergencyAccessDesc": {
     "message": "Grant and manage emergency access for trusted contacts. Trusted contacts may request access to either View or Takeover your account in case of a emergency. Visit our help page for more information and details into how zero knowledge sharing works."
```

Comment:
While you're in here, can you please correct this one too?
> your account in case of a emergency.

Should be
> your account in case of **an** emergency.